src/App.jsx

```diff
@@ -1,17 +1,40 @@
 import './App.css';
+import messages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+import { useState } from 'react';
+
+
+function App() {
+  const [chatData, setChatData] = useState(messages);
+  const [likesCount, setLikesCount] = useState(0);
```

Comment:
Do we necessarily need this piece of state? Couldn't we just use the fact that if a message is liked then we could have a function that calculates the total number of likes by iterating over the messages that is `chatData`? This works each render since each time a message is liked `chatData` state is changed.